apps/api/src/controllers/applicationController.ts

```diff
@@ -173,3 +173,9 @@ export const approveApplication = async ({ applicationId }: ApproveApplication):
 		return failure(message, error);
 	}
 };
+
+export const rejectApplication = async ({ applicationId }: { applicationId: number }) => {
+	const database = getDbInstance();
+	const service: ApplicationService = applicationSvc(database);
+	return await service.rejectApplication({ applicationId });
+};
```

Comment:
Related to discussion here: https://github.com/Pan-Canadian-Genome-Library/daco/pull/179#pullrequestreview-2607042602

Update `rejectApplication` to follow the example in `approveApplication`

`rejectApplication`	should get application by Id, create new ApplicationStateManager, then call `stateManager.dacReject()` (just added)

`dacReject` covers DB methods of updating the Application record & creating an Action record